.github/workflows/on-merge.yml

```diff
@@ -26,6 +26,7 @@ jobs:
         with:
           nexus_username: ${{ secrets.NEXUS_USERNAME }}
           nexus_password: ${{ secrets.NEXUS_PASSWORD }}
+          force_signature: true
```

Comment:
This is not something we want, it just takes 15 minutes to crash

https://github.com/Jahia/javascript-modules/actions/runs/17767129016/job/50493231898?pr=496


```suggestion
```